api/tests/unit/domain/usecases/create-user-from-external-identity-provider_test.js

```diff
@@ -93,135 +89,22 @@ describe('Unit | UseCase | create-user-from-external-identity-provider', functio
     });
   });
 
-  context('when user comes from Pole Emploi', function () {
-    it('should call createUserAccount method to return user id and id token', async function () {
+  context('when required properties are not returned by external API', function () {
+    it('should raise an error and log details if required properties are not returned by external API', async function () {
```

Comment:
cette phrase est un peu repetitif du context.. je verrai qq ch de plus simple, du genre
`should raise an error and log details`